server/lib/nl/detection/date.py

```diff
@@ -35,13 +35,18 @@
     r'(in|after|on|before|since|by|until|from|between) (\d{4})(?:,)? (Jan|Feb|Mar|Apr|May|Jun|Jul|Aug|Sep|Oct|Nov|Dec)',
 ]
 
+# prep to use for LAST_YEAR dates which should include the current year.
+_LAST_YEARS_PREP = 'last_years'
```

Comment:
nit: mention why the `_`?

Reply:
I was puzzled by the `_` in `last_years`, but later realized these are sentinels you use to indicate things rather than anything else, right?  (Perhaps an enum for these might be more readable)

Reply:
ah yes, this is just a placeholder to use as the "prep" for all the dates that are last x years. This is the only case of using a placeholder prep, so maybe I can make that more clear in the comments & can switch to an enum if there are more cases of this